.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
+  - init: go get && go build ./... && go test ./...
+    command: go run
```

Comment:
@edmooney any reason for adding this gitpod configuration?

Reply:
@vinamra28 - accident

Reply:
it seems that this file is still present 😅